### PR TITLE
2022-10-06-BSC-Token-Hub

### PR DESCRIPTION
**Pull request by JediFaust**
BSC Token Hub exploit with over $120 Million funds lost

### CODE REVIEW

**Comment by JediFaust** [comment]
So, It needs to pass every statement? And facts should be verifiable from sources?

**Comment by evgenydmitriev** [comment]
> So, It needs to pass every statement? And facts should be verifiable from sources?

Sometimes, the bot gets things wrong. Someone from the wiki team will go through everything manually. That said, you can help the bot and the team by adding markdown links directly in the failed claims.

**Comment by JediFaust** [comment]
Sources have been added, for the failed claims, and some details omitted. When this PR will be merged, I would continue on other incidents 👏🏻

File: content/attacks/posts/2022-10-06-BSC-Token-Hub.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: "BSC Token Hub Hit By $586 Million Bridge Hack"
+---
+
+## Summary
+
+On October 6, 2022, BSC Token Hub, a bridge between BNB Beacon Chain (BEP2) and Binance Smart Chain (BEP20) was exploited. The native cross-chain bridge between BNB Beacon Chain (BEP2) and BNB Smart Chain (BEP20), also known as BNB Token Hub was exploited. The hacker used a low-level proof vulnerability and 2,000,000 $BNB were drained out of thin air. Consequently, the hacker began [bridging the funds to Fantom and Ethereum chains](https://www.coinbase.com/blog/bsc-token-hub-compromise-investigation-and-analysis-8). [The security experts in collaboration with validators were able to save the majority of the funds](https://www.bnbchain.org/en/blog/bnb-chain-ecosystem-update). The [hacker managed to bridge 127,000,000 $USD using AnySwap and Stargate bridges](https://rekt.news/bnb-bridge-rekt), with 53% of the stolen funds going to Ethereum, 33% to Fantom, and the rest to other chains. The remaining assets were left frozen in the attacker's address. 
+
+## Attacker
+
+Attackers address on BSC: 0x489a8756c18c0b8b24ec2a2b9ff3d4d447f79bec
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-10-06-BSC-Token-Hub.md:14]: is there any known info about identity of the hackers?

File: content/attacks/posts/2022-10-06-BSC-Token-Hub.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Losses
+
+2 million BNB was minted, it equals $586 million. Only $127 million worth of assets were bridged, and the rest is left frozen.
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-10-06-BSC-Token-Hub.md:18]: can you link a source for this one?

Comment (JediFaust) [reply content/attacks/posts/2022-10-06-BSC-Token-Hub.md:18]: https://www.bitget.com/academy/BNB-Chain-586-Million-dollar-Hack-The-Horror-Before-Halloween
Sure, here it is, amounts may differ a little, cause of various token prices at the moment of calculations

File: content/attacks/posts/2022-10-06-BSC-Token-Hub.md (hunk 1 above)
Comment (JediFaust) [reply content/attacks/posts/2022-10-06-BSC-Token-Hub.md:14]: Just this address, in various chains. I may include information for each chain, and the funds in them, if it's preferred

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
@JediFaust, could you run your article through a spellchecker? I recommend [LanguageTool](https://languagetool.org/).

**Comment by JediFaust** [comment]
@evgenydmitriev Done 👍🏻

**Comment by JediFaust** [comment]
There is links to the sources inside the report, covering most failed claims

**Review by jhirschkorn — APPROVED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**